doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -293,6 +293,96 @@ The following table indicates the software maturity levels of the support for Go
         - --
         - --
 
+The following table indicates the software maturity levels of the support for each Fast Pair feature:
+
+.. toggle::
+
+   .. _software_maturity_fast_pair_feature:
+
+   .. list-table:: Google Fast Pair feature support
+      :header-rows: 1
+      :align: center
+      :widths: auto
+
+      * -
```

Comment:
`Feature`?

Reply:
I didn't add it so it looks the same as other feature tables. Should I add it?

Please take a look at the Protocol and Security feature tables.

Reply:
We can leave it as is then

Reply:
I can add it if others agree that it is a step in the right direction

Reply:
yeah, we could leave it.